concent_api/core/models.py

```diff
@@ -497,13 +497,15 @@ def clean(self) -> None:
             deserialized_report_computed_task = deserialize_message(self.report_computed_task.data.tobytes())  # pylint: disable=no-member
 
         # If available, the report_computed_task nested in force_get_task_result must match report_computed_task.
-        if (
-            self.force_get_task_result is not None and
-            deserialize_message(self.force_get_task_result.data).report_computed_task != deserialized_report_computed_task
-        ):
-            raise ValidationError({
-                'force_get_task_result': "ReportComputedTask nested in ForceGetTaskResult must match Subtask's ReportComputedTask."
-            })
+        if self.force_get_task_result is not None:
+            if isinstance(self.force_get_task_result.data, bytes):
```

Comment:
why this new `if ... else`? When it would be possible for `self.force_get_task_result.data` not to be `bytes`?

Reply:
Because there are some cases when `self.force_get_task_result.data` could be `StoredMessage` object instead of `bytes`. We have same situation with `self.report_computed_task.data`